terraform/google_ca.tf

```diff
@@ -14,9 +14,6 @@ resource "google_privateca_ca_pool" "default" {
       allow_config_based_issuance = true
     }
   }
-  lifecycle {
```

Comment:
Is this removal intentional? If it is, this might warrant a comment stating why we do not prevent accidental deletions.

Reply:
added it back, probably from testing it was removed.